etc/profile-m-z/quodlibet.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for quodlibet
+# Description: Music player and music library manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quodlibet.local
+# Persistent global definitions
+include globals.local
+
```

Comment:
Add:
noblacklist ${HOME}/.cache/quodlibet
noblacklist ${HOME}/.config/quodlibet
noblacklist ${HOME}/.quodlibet

The first two need to be added as blacklists to disable-programs.inc (blacklist ${HOME}/.quodlibet is aleady in that file).